### PR TITLE
Release v0.52.0

### PR DESCRIPTION
**Pull request by seanchen1991**
Closes: #XXX

## Description

______

### PR author checklist:
- [ ] Added changelog entry, using [`unclog`](https://github.com/informalsystems/unclog).
- [ ] Added tests.
- [ ] Linked to GitHub issue.
- [ ] Updated code comments and documentation (e.g., `docs/`).
- [ ] Tagged *one* reviewer who will be the one responsible for shepherding this PR.

### Reviewer checklist:

- [ ] Reviewed `Files changed` in the GitHub PR explorer.
- [ ] Manually tested (in case integration/unit/mock tests are absent).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,84 @@
 # CHANGELOG
 
+*April 26, 2024*
+
+## v0.52.0
```
Comment (rnbguy) [inline CHANGELOG.md]: The date should be after the header.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,84 @@
...
+
+This release adds some exciting changes, improvements, and new features to ibc-rs.
+First and foremost, support for the IBC protocol's [client recovery][client-recovery]
+mechanism has been implemented, which provides a route for frozen and expired IBC clients
+to be re-instated following a successful governance vote. In addition, a new crate,
+`ibc-client-cw`, facilitates CosmWasm contract creation for light clients built using
```
Comment (rnbguy) [inline CHANGELOG.md:11]: Can you please also add a line that we are also compiling ics07 as cw-contract as well?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,84 @@
...
+`ibc-rs`.
+
+This release also includes a myriad of other bug-fixes and improvements,
+such as enhancing the portability of ibc-rs's Validation and Execution Context traits,
+as well as fixing an incompatibility with how ibc-rs parses `PrefixDenom`s compared
+to ibc-go, among many others.
+
+Finally, the minimum-supported Rust version has been updated to `1.72`.
```
Comment (rnbguy) [inline CHANGELOG.md]: Also mention about `tendermint` and `ibc-proto` dependency bump.

Comment (rnbguy) [reply CHANGELOG.md]: Also, that the new `ibc-derive` version is `0.7.0`.

**Review by rnbguy — CHANGES_REQUESTED** [review]
Added a few comments on the release summary.

File: CHANGELOG.md (hunk 3 above)
Comment (seanchen1991) [reply CHANGELOG.md]: [5474db7](https://github.com/cosmos/ibc-rs/pull/1199/commits/5474db7597c8cd498c65d1d06960aeb7b0be38a1)

File: CHANGELOG.md (hunk 2 above)
Comment (seanchen1991) [reply CHANGELOG.md:11]: [5474db7](https://github.com/cosmos/ibc-rs/pull/1199/commits/5474db7597c8cd498c65d1d06960aeb7b0be38a1)

File: CHANGELOG.md (hunk 1 above)
Comment (seanchen1991) [reply CHANGELOG.md]: [5474db7](https://github.com/cosmos/ibc-rs/pull/1199/commits/5474db7597c8cd498c65d1d06960aeb7b0be38a1)

**Review by rnbguy — APPROVED** [review]
Looks great :raised_hands:

Ready for shipping :package: Thanks, Sean :pray:

**Pull request merged**